### PR TITLE
Request: NBB Logo

### PR DESCRIPTION
**Pull request by thorbenw**
**Issue:** none

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`
  - [x] I read [CONTRIBUTING.md](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) carefully
  - [x] I created a preview image (see below) using the [MDI-Sandbox](https://petershaggynoble.github.io/MDI-Sandbox/simpleicons/preview/)
  - [x] I didn't find [any other request](https://github.com/simple-icons/simple-icons/pulls?page=2&q=is%3Apr+is%3Aopen) for this brand
  - [x] I successfully built and ran the website in [Gitpod](https://gitpod.io)
  - [x] I fixed what made the lint job fail

### Description
  - All required information (e.g. brand colour) has been gathered from the official NBB [Style Guide](https://presse.notebooksbilliger.de/presskits/style-guide)
  - I used an svg version of the logos presented in the style guide (only png versions available there, svg version included in [logo_nbb_ohnecom-weiss.zip](https://github.com/simple-icons/simple-icons/files/4455478/logo_nbb_ohnecom-weiss.zip)) as the starting point for creating the optimized svg file
  - The current primary url is https://www.notebooksbilliger.de (Alexa rank around [10k](https://www.alexa.com/siteinfo/notebooksbilliger.de)), but the whole page is undergoing a redesign, that's why I'm already using the new logo (see outdated logo [here](https://github.com/notebooksbilliger)) and the new brand name

nbb

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
Welcom to Simple Icons, @thorbenw and thanks for the contribution :+1:

**Alexa rank:** [~10.5k](https://www.alexa.com/siteinfo/notebooksbilliger.de)

> I can't attach it because svg isn't supported here

You can drop it into a zip file and upload that.

From their website, it looks like the name should be `notebooksbilliger.de`, rather than `NBB`.

Is [this logo](https://prowly-uploads.s3.eu-west-1.amazonaws.com/uploads/4933/assets/57629/nbb_logo_2000.png) part of the old branding? If not, I wonder could we get away with using the arrow from that instead, as seen in their favicon, a few places on their website and on some of the signage in their shops. It seems to me as though that might be more recognisable but, not being familiar with the brand, I'd like to hear from some of the German members of the @simple-icons/maintainers team before making that change.

**Comment by thorbenw** [comment]
Hello @PeterShaggyNoble and thanks for your reply 👍

I've uploaded the svg in a zip file as you suggested :relaxed:

The Primary url will be changed to [nbb.com](http://www.nbb.com) as part of the rebranding.

Yes, [this logo](https://prowly-uploads.s3.eu-west-1.amazonaws.com/uploads/4933/assets/57629/nbb_logo_2000.png) belongs to the outdated branding. The 'arrow-screen' logo will soon be abandoned, and therefore the favicon will also be replaced (indeed with the icon I put in here, but in ico format and being non-monochromatic).

@runxel / all other german @simple-icons/maintainers I work at nbb, so feel free to ask anything you like and I'll be happy to clarify anything you need 📝

**Comment by thorbenw** [comment]
Hi @runxel ,

thanks for approving :thumbsup:

The current url notebooksbilliger.de will remain ours, but we'll redirect to nbb.com then (currently, it's the opposite, we redirect from nbb.com to notebooksbilliger.de). We're curently working on the changes in AD (derive UPNs from nbb.com and the like), and once we've finished that we'll reuse/adjust the certificate for nbb.com (if you go to https://nbb.com Right now you'll still get cert errors) and then start changing the primary url in all websites.
In other words, I'll need to file another PR then to adjust the url in `_data/simple-icons.json` :joy:

**Comment by runxel** [comment]
Okay, great! I'm cool with this. :) Everbody else, too @simple-icons/maintainers?
Thanks again for the PR!

**Comment by PeterShaggyNoble** [comment]
> I work at nbb

Ah, that changes things, so - your word is gospel on this! 😆

Given that you're in the process of rebranding, I'd personally prefer to hold off on merging this until that is complete. Having said that, our next release isn't scheduled until the 25th so, if you plan to have the rebranding completed by then, we can go ahead and merge now.

**Comment by thorbenw** [comment]
Hi @PeterShaggyNoble ,

I see - I'd say the new favicon is a good trigger (changing the primary url might still take some time)? I can monitor the progress of the new-favicon tickets and let you know when they're done and the new favicon is visible (and the old one has gone).

**Comment by PeterShaggyNoble** [comment]
That seems reasonable to me, @thorbenw.

Note, though, that waiting to merge is just my personal preference. If you and your company are happy for this to be merged now then that's fine.

**Comment by thorbenw** [comment]
@PeterShaggyNoble let's just wait and see if any changes still need to be made to use this logo as a favicon :blush:

**Comment by PeterShaggyNoble** [comment]
Fair enough, @thorbenw :+1: Keep us posted and we'll merge this on your say-so.

**Comment by thorbenw** [comment]
@PeterShaggyNoble  the Favicon has now been published, at least on our mobile site https://m.notebooksbilliger.de

Merge would be ok for us now.

**Comment by PeterShaggyNoble** [comment]
Thanks for letting us know, @thorbenw, and for your work on this one.

Unfortunately, we just missed getting this out in yesterday's release but the good news is that our new automated release process has allowed us to increase the frequency of our releases from fortnightly to weekly so this will be available on Sunday.

**Comment by thorbenw** [comment]
@PeterShaggyNoble that's great news, thanks!
